robots.json

```
@@ -20,6 +20,13 @@
         "frequency": "No information provided.",
         "description": "Scrapes data for AI systems."
     },
+    "Andibot": {
+        "operator": "[Andi](https://andisearch.com/)",
+        "respect": "Unclear at this time",
+        "function": "search engine using generative AI, AI Search Assistant",
```

```suggestion
        "function": "Search engine using generative AI, AI Search Assistant",
```

Let's adjust the casing here.